src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java

```diff
@@ -243,29 +175,6 @@ protected NewUserCreationRequest createUserRequest(List<String> userRoles) {
         return userCreationRequest;
     }
 
-
-    public String getExternalSuperUserTokenWithRetry(String email, String firstName, String lastName) {
-        int counter = 0;
-
-        String idamResponse = idamOpenIdClient
-                .getExternalOpenIdTokenWithRetry(superUserRoles(), firstName, lastName, email);
-
-        while (idamResponse.equalsIgnoreCase("504") && counter < 4) {
```

Comment:
I understand that this piece of code is deleted, but if we reintroduce the code, can we use a retry mechanism rather than a raw implementation?
We can discuss if there are any challenges.

Reply:
I'm sure we can do a mechanism rather than raw implementation - this was done as quick fix for failing FTs. However, these methods are no longer used so have been deleted.

Reply:
Thanks